src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/keybind/KeyBindHandler.kt

```diff
@@ -4,25 +4,39 @@ import com.chattriggers.ctjs.minecraft.wrappers.World
 import net.minecraftforge.common.MinecraftForge
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 import net.minecraftforge.fml.common.gameevent.TickEvent
+import java.util.concurrent.CopyOnWriteArrayList
 
 internal object KeyBindHandler {
     init {
         MinecraftForge.EVENT_BUS.register(this)
     }
 
-    private val keyBinds = mutableListOf<KeyBind>()
+    private val keyBinds = CopyOnWriteArrayList<KeyBind>()
 
     fun registerKeyBind(keyBind: KeyBind) {
         keyBinds.add(keyBind)
     }
 
+    fun unregisterKeyBind(keyBind: KeyBind) {
+        keyBinds.remove(keyBind)
+    }
+
     fun clearKeyBinds() = keyBinds.clear()
 
+    fun getKeyBinds() = keyBinds
```

Comment:
This can probably be internal since JS shouldn't need this

Reply:
The whole object is internal already.

Reply:
you right you right